### DISCUSSION TITLE
EPROM flash adapters?

### DISCUSSION THREAD

**Discussion by Aerik72** [sanni/cartreader#358] [Need help]
I'm thinking about putting together a project to translate some of my Famicom & Super Famicom games into English by flashing EPROMs to install into the original cartridges. It would really cut down the cost of the project if I could use the cart reader to also flash my ICs. (Specifically, I'll need to write: 27C512, 29F800, 29F016.)

I see help pages in the wiki for how to use the software to flash ROMs, but they show different adapter PCBs and sockets that I can't find any other references to in the PCB schematics or parts lists.

How can I build some of those adapters? Are they part of this github repo, or are they from elsewhere?

Thanks for any help! I'm excited to dig into this build. :)

**Comment by sanni** [comment]
I have to redo the flash adapters, didn't have the time yet. The pin-out changed slightly with V4 and the old PCBs are not compatible anymore.

https://web.archive.org/web/20201127171309/https://github.com/sanni/cartreader/wiki/Flash--and-Eprom-Adapter
https://github.com/sanni/cartreader/tree/bd3eaa106b6e4cbdffbfa89a3fb5fedf3e028b17/pcb/adapters

**Reply by Aerik72** [reply]
Ah, I understand. I can always get myself a FlashcatUSB XPORT to do it.

No worries, thank you very much for the response!

**Reply by herzmx** [reply]
Hi @sanni i check the pinout.ods file and only found a change in OE(PH1) for MX26L6420 adapter that now it's conected to conected to CLK1 in the SNES port, so i cut OE from adapter and put dupont wire to D16 in six slot slot adapter to OE in adapter. Afther that the EPROM it's recognized when select MX26L6420 adapter in flash menu.

But when try to erase or write the program it's keep in loop of busyCheck.

Do you have a clue about someting i missing?

Thanks...
